src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+    print('\n')
+
+    print('Porcentaje de valores null:')
+    print(df.isnull().mean() * 100)
+    print('\n')
+```
+
+
+```python
+quick_view(df_orders)
+```
+
+    Primeras filas:
+                   id                                            user_id  \
+    10  2204073066628  62e271062eb827e411bd73941178d29b022f5f2de9d37f...   
+    20  2204707520644  bf591c887c46d5d3513142b6a855dd7ffb9cc00697f6f5...   
+    21  2204838822020  329f08c66abb51f8c0b8a9526670da2d94c0c6eef06700...   
+    34  2208967852164  f6451fce7b1c58d0effbe37fcb4e67b718193562766470...   
+    49  2215889436804  68e872ff888303bff58ec56a3a986f77ddebdbe5c279e7...   
+    
+                created_at order_date  user_order_seq  \
+    10 2020-04-30 14:32:19 2020-04-30               1   
+    20 2020-04-30 17:39:00 2020-04-30               1   
+    21 2020-04-30 18:12:30 2020-04-30               1   
+    34 2020-05-01 19:44:11 2020-05-01               1   
+    49 2020-05-03 21:56:14 2020-05-03               1   
+    
+                                            ordered_items  
+    10  [33618849693828, 33618860179588, 3361887404045...  
+    20  [33618835243140, 33618835964036, 3361886244058...  
+    21  [33618891145348, 33618893570180, 3361889766618...  
+    34  [33618830196868, 33618846580868, 3361891234624...  
+    49  [33667166699652, 33667166699652, 3366717122163...  
+    
+    
+    Información general:
+    <class 'pandas.core.frame.DataFrame'>
+    Index: 8773 entries, 10 to 64538
+    Data columns (total 6 columns):
+     #   Column          Non-Null Count  Dtype         
+    ---  ------          --------------  -----         
+     0   id              8773 non-null   int64         
+     1   user_id         8773 non-null   object        
+     2   created_at      8773 non-null   datetime64[us]
+     3   order_date      8773 non-null   datetime64[us]
+     4   user_order_seq  8773 non-null   int64         
+     5   ordered_items   8773 non-null   object        
+    dtypes: datetime64[us](2), int64(2), object(2)
+    memory usage: 479.8+ KB
+    
+    
+    Número de valores null:
+    id                0
+    user_id           0
+    created_at        0
+    order_date        0
+    user_order_seq    0
+    ordered_items     0
+    dtype: int64
+    
+    
+    Porcentaje de valores null:
+    id                0.0
+    user_id           0.0
+    created_at        0.0
+    order_date        0.0
+    user_order_seq    0.0
+    ordered_items     0.0
+    dtype: float64
+    
+    
+
+
+Vemos que no hay valores nulos y que parece que trabjamos con una muestra de un dataset mayor: 8773 entries, 10 to 64538. Todo parece OK aunque no sé bien como hacer el match entre ordered_items y el variant_id de los otros datasets. Se me ocurre hacer otra columna con el espacio temportal entre created_at y order_date para obtener aún mas datos significativos como que productos generan mas dudas al consumidor, compras impulsivas tipos de consumidores, etc...
```

Comment:
_trabjamos con una muestra de un dataset mayor: 8773 entries, 10 to 64538._ -> what does it mean? it's confusing

Reply:
try writing with a minimum level of orthography (_mas_-> _más_, _que productos_- >_qué productos_...)
also if you use the name of columns do it with format:  `created_at` or `order_date`